plugins/renepay/mcf.c

```diff
@@ -501,15 +502,18 @@ static bool linearize_channel(const struct pay_parameters *params,
 
 	capacity[0]=a;
 	cost[0]=0;
+	assert(params->base_probability > 5e-7);
```

Comment:
I should add a check here for base probability to be no greater than 1, otherwise could get negative cost channels.